File: AC/ABS/abc083/b.go
package main

import "fmt"

func main() {
	var n, a, b, tmp, dsum, sum int
	fmt.Scan(&n, &a, &b)
	for i := 1; i <= n; i++ {
		tmp = i
		dsum = 0
		for tmp != 0 {
			dsum += tmp % 10
			tmp /= 10
		}
		if a <= dsum && dsum <= b {
			sum += i
		}
	}
	fmt.Println(sum)
}

File: AC/ABS/abc083/b.py
n, a, b = map(int, input().split())
sum = 0
for i in range(1, n+1):
    dsum = 0
    tmp = i
    while tmp != 0:
        dsum += tmp % 10
        tmp //= 10
    if a <= dsum and dsum <= b:
        sum += i
print(sum)
